doc/sphinx/changes.rst

```diff
@@ -277,6 +277,13 @@ Tactics
   (`#15245 <https://github.com/coq/coq/pull/15245>`_,
   fixes `#15244 <https://github.com/coq/coq/issues/15244>`_,
   by Hugo Herbelin).
+- **Deprecated:**
+  the :tacn:`bfs eauto` tactic. Since its introduction
+  it was behaving exactly like the normal tacn:`eauto` tactic.
+  Use instead the `bfs` flag of the :tacn:`typeclass eauto` tactic
```

Comment:
```suggestion
  it has behaved exactly like the tacn:`eauto` tactic.
  Use :tacn:`typeclasses eauto` with the `bfs` flag instead
```

Reply:
Why `has behaved` instead of just `behaved`?

Reply:
Or actually `behaves` since it's not removed yet

Reply:
"has behaved" = behavior existed in the past and continues to the present
"behaved" = behavior existed in the past, doesn't say it continues to the present
"behaves" = only talks about the present but "Since its introduction" requires a past tense

If you drop "Since its introduction" then "behaves" is OK.

You could

Reply:
The present perfect tense in English implies something is still affecting reality. It makes sense to use it there.

Reply:
Also, if it behaves "exactly like the tacn:`eauto` tactic.", why recommend using `typeclasses eauto` instead?

Reply:
If you want a bfs behaviour, that is.

Reply:
Ah, because it has never worked.